OrbitQt/orbitmainwindow.cpp

```diff
@@ -1165,11 +1173,38 @@ void OrbitMainWindow::SetTarget(const orbit_qt::StadiaTarget& target) {
   app_->SetGrpcChannel(connection->GetGrpcChannel());
   app_->SetProcessManager(target.GetProcessManager());
   app_->SetTargetProcess(target.GetProcess());
+
   // TODO (b/173011168): Color should not be defined inline. Replace with defined "good color"
   // (green).
-  target_label_->setStyleSheet("#TargetLabel { color: #66BB6A; }");
+  static const QString kTargetLabelDefaultStyleSheet = "#TargetLabel { color: #66BB6A; }";
+  target_label_->setStyleSheet(kTargetLabelDefaultStyleSheet);
   target_label_->setText(QString::fromStdString(target.GetProcess()->name()) + " @ " +
                          target.GetConnection()->GetInstance().display_name);
+
+  using ProcessInfo = orbit_grpc_protos::ProcessInfo;
+  target.GetProcessManager()->SetProcessListUpdateListener([&](std::vector<ProcessInfo> processes) {
+    // This lambda is called from a background-thread, so we use QMetaObject::invokeMethod
+    // to execute our logic on the main thread.
+    QMetaObject::invokeMethod(this, [&, processes = std::move(processes)]() {
+      const auto is_current_process = [this](const auto& process) {
+        const auto target_process = app_->GetTargetProcess();
```

Comment:
I have been scolded for using `auto` too much, so I'll do the same here :-) Same for the `auto`s following below
https://google.github.io/styleguide/cppguide.html#Type_deduction

Reply:
Will change that! :+1:

Reply:
done. I just like auto too much. :see_no_evil: Time to move on to C++20!